src/sass/modules/modals.scss

```diff
@@ -37,13 +36,7 @@
 }
 
 .modal-cancel {
```

Comment:
if this is all that's on this style I think we can use `u-margin--r-8`, but is there no hover/focus style at all for these cancels?

Reply:
I'll check with Mark. Seems like there should be.